Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UwpAppxBuildTools.cs

```diff
@@ -163,33 +163,42 @@ private static async Task<int> Run(string fileName, string args, bool showDebug,
 
         private static async Task<string> FindMsBuildPathAsync()
         {
-            var result = await new Process().StartProcessAsync(
+            // Finding msbuild.exe involves different work depending on whether or not users
+            // have VS2017 or VS2019 installed.
+            foreach (VSWhereFindOption findOption in VSWhereFindOptions)
+            {
+                var result = await new Process().StartProcessAsync(
                 new ProcessStartInfo
                 {
                     FileName = "cmd.exe",
                     CreateNoWindow = true,
                     UseShellExecute = false,
                     RedirectStandardOutput = true,
                     RedirectStandardError = true,
-                    Arguments = $@"/C vswhere -all -products * -requires Microsoft.Component.MSBuild -property installationPath",
+                    Arguments = findOption.arguments,
```

Comment:
If we pass this an argument it doesn't support (`-find`), does it error or just not find anything?

Reply:
It reports an error back through stdout from what I remember